scene/resources/packed_scene.cpp

```diff
@@ -92,6 +92,8 @@ Node *SceneState::instance(GenEditState p_edit_state) const {
 
 		if (i > 0) {
 
+			ERR_EXPLAIN("Object " + snames[n.name] + " in escn file is missing the parent node.")
```

Comment:
This code is not specific to `escn` though, so I guess any scene file without `parent` tag would trigger this error?

Reply:
I assume so, but I haven't tested with other file formats. I'll remove `escn` from the error message.